### PR TITLE
Add strings up to 6622.

### PR DESCRIPTION
**Pull request by CsyeCokTheSolly**
NOTE: The lines are not translated yet, but that may change overtime.

Applying for issue(s):
- #2678
- #2679
- #2692
- #2720
- #2732
- #2755
- #2769

### CODE REVIEW

File: data/language/uk-UA.txt [hunk 1]
```diff
@@ -3695,6 +3695,36 @@ STR_6589    :Показати кнопки вікна зліва
 STR_6590    :Показати кнопки вікна (напр. для закриття вікна) ліворуч від рядка заголовка.
 STR_6591    :Не можна звільнити поки працівник лагодить атракціон.
 STR_6592    :Не можна звільнити поки працівник тестує атракціон.
+STR_6593    :Remove park fences
```
Comment (Gymnasiast) [inline data/language/uk-UA.txt:3698]: Newly added string need to be translated. If you don’t have time now, just remove them - the game will still display them correctly.

Comment (CsyeCokTheSolly) [reply data/language/uk-UA.txt:3698]: Alr i got them translated.
I'll try to get it to this PR.

Comment (Gymnasiast) [reply data/language/uk-UA.txt:3698]: It looks like you forgot STR_6593 - that one is still in English.

Comment (CsyeCokTheSolly) [reply data/language/uk-UA.txt:3698]: oops

Comment (CsyeCokTheSolly) [reply data/language/uk-UA.txt]: Fixed. I hope

**Pull request merged**